### DISCUSSION TITLE
Android feasibility

### DISCUSSION THREAD

**Discussion by jackf723** [dectalk/dectalk#20] [Ideas]
I feel like now is as good a time as any to throw the idea out there given the recent Pixel7 happenings. With Eloquence out of the equation for subsequent new Android phones, many are once again on the lookout for a good, responsive formant synth for Android. If anyone around here knows their way around the Android TTS framework, would the current code for Linux provide a good starting point towards an Android port?

**Comment by pbsinnett** [comment]
I would love this. Although I will keep suggesting Eloquence to whoever I can at Google, this synth would be amazing on Android. DT was my first synth back in '96. If it did come to Android, hopefully it would support emoji.
